src/diskquota.c

```diff
@@ -78,6 +78,7 @@ int  diskquota_max_workers             = 10;
 int  diskquota_max_table_segments      = 0;
 int  diskquota_max_monitored_databases = 0;
 int  diskquota_max_quota_probes        = 0;
+time_t diskquota_hashmap_overflow_report_timeout = 0;
```

Comment:
why not int type?

Reply:
Because timestamp can be bigger than int. I will replace it with TimestampTz.

Reply:
GUC defines int in seconds, while you mau use it as 1000 * GUC milliseconds

Reply:
I don't think anyone will use such a large timeout. I can limit the GUC as `INT_MAS / 1000`